apache2/re.c

```diff
@@ -2513,6 +2513,11 @@ static void msre_perform_disruptive_actions(modsec_rec *msr, msre_rule *rule,
             }
         }
     }
+    if (actionset->intercept_action_rec->metadata->type == ACTION_DISRUPTIVE) {
```

Comment:
I don't think so...?  Here's my understanding:

The loop before this line looks through all the actions in `actionset->actions` and run the execute callback for each.  For some reason, though, it doesn't seem like the `actionset->intercept_action_rec` is part of the list in `actionset->actions` -- so I added a few lines to check whether `actionset->intercept_action_rec` is an intercept action, and if so to run the execute callback for it.